backend/entity/Cargo.toml

```diff
@@ -4,3 +4,4 @@ version.workspace = true
 edition = "2021"
 
 [dependencies]
+sea-orm = "1.1.4"
```

Comment:
Please use [`workspace-dependencies`](https://doc.rust-lang.org/cargo/reference/workspaces.html#the-dependencies-table), so we don't have to update version in 2 places